index.html

```diff
@@ -904,7 +904,7 @@ <h4>Useful Links</h4>
               <li><i class="bi bi-chevron-right"></i> <a href="#">About us</a></li>
               <li><i class="bi bi-chevron-right"></i> <a href="#">Services</a></li>
               <li><i class="bi bi-chevron-right"></i> <a href="#">Terms of service</a></li>
-              <li><i class="bi bi-chevron-right"></i> <a href="#">Privacy policy</a></li>
+              <li><i class="bi bi-chevron-right"></i> <a href="privacy-policy.html">Privacy policy</a></li>
```

Comment:
@basiima we may need to first create `privacy-policy.html` before adding routes to it...i mean why are we adding routes to pages that dont exist?